### PR TITLE
EZP-30665: Custom styles dropdown becomes unusable when contains many…

### PR DESCRIPTION
**Pull request by czechmarcin**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-30665
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

It is not possible to select custom style when having configured many of them.

PR result:
Screenshot 2019-06-13 at 12 55 34

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/scss/_alloyeditor-custom-styles.scss [hunk 1]
```diff
@@ -20,3 +20,7 @@
         font-family: courier, monospace;
     }
 }
+ul.ae-listbox {
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_alloyeditor-custom-styles.scss]: Those styles are not related to custom styles. It will affect all `lists` in AE so I would move this to the `ezplatform-admin-ui/src/bundle/Resources/public/css/alloyeditor/alloyeditor-ez.css`
Also, I would change the selector to match AE selector: `.ae-ui .ae-dropdown .ae-listbox`

Comment (czechmarcin) [reply src/bundle/Resources/public/scss/_alloyeditor-custom-styles.scss]: @dew326 Please have a look now. I used and tested your proposition.

**Comment by czechmarcin** [comment]
> On Firefox & Safari (not on Chrome & Edge) second scrollbar appears at the bottom.
> Question is can we do something about it at a low cost? It's not a big deal, just looks bad.
> Screenshot 2019-06-13 at 15 35 06

I'm not sure if that's the case here. @dew326, do you think it should be changed?

**Comment by dew326** [comment]
@czechmarcin We discussed this issue before @micszo put the comment. It would be good to fix this.

**Comment by czechmarcin** [comment]
@micszo As I'm not able to reproduce it even on the same versions of web browsers and the customer is waiting for the fix, could we agree to proceed with existing changes without fixing the width? Cc: @dew326

**Comment by czechmarcin** [comment]
@micszo After a deeper investigation, I am able to reproduce this issue. Working on a fix

**Comment by dew326** [comment]
@czechmarcin can you merge it up?

**Comment by czechmarcin** [comment]
@dew326 merged
